site-src/concepts/tooling.md

```diff
@@ -6,18 +6,18 @@ Gateway API has tooling to facilitate interacting with the resources.
 
 Interested in seeing what your ingress resources will look like in Gateway API? `ingress2gateway` provides an easy manner to translate provider-specific resources to Gateway API resources. Managed by the Gateway API SIG-Network subproject.
 
-Get started [here](https://github.com/kubernetes-sigs/ingress2gateway?tab=readme-ov-file#installation)!
+Get started with [kubernetes-sigs/ingress2gateway: installation](https://github.com/kubernetes-sigs/ingress2gateway?tab=readme-ov-file#installation)!
```

Comment:
I've added `: installation` to distinguish from the cases where links are to the general readme...